osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Online.API.Requests.Responses;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using osu.Game.Graphics.Sprites;
+using osuTK.Graphics;
+
+namespace osu.Game.Overlays.Changelog
+{
+    public class ChangelogBuild : FillFlowContainer
+    {
+        public Action<APIChangelogBuild> SelectBuild;
+
+        protected readonly APIChangelogBuild Build;
+
+        public readonly FillFlowContainer ChangelogEntries;
+
+        public ChangelogBuild(APIChangelogBuild build)
+        {
+            Build = build;
+
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+            Direction = FillDirection.Vertical;
+            Padding = new MarginPadding { Horizontal = 70 };
+
+            Children = new Drawable[]
+            {
+                CreateHeader(),
+                ChangelogEntries = new FillFlowContainer
```

Comment:
I suggest to create a private method called `CreateList` or similar.

Reply:
why?

Reply:
you mean to move the parsing logic i haven't yet refactored? this PR is not ready for merge just yet.